pkg/resources/dynamic_table.go

```diff
@@ -75,6 +75,34 @@ var dynamicTableShema = map[string]*schema.Schema{
 		Optional:    true,
 		Description: "Specifies a comment for the dynamic table.",
 	},
+	"refresh_mode": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "INCREMENTAL if the dynamic table will use incremental refreshes, or FULL if it will recompute the whole table on every refresh.",
+		ForceNew:    true,
+		DiffSuppressFunc: func(_, oldValue, newValue string, d *schema.ResourceData) bool {
+			switch {
+			case oldValue == "INCREMENTAL" && newValue == "FULL":
+				return false
+			case oldValue == "FULL" && newValue == "INCREMENTAL":
+				return false
+			case oldValue == "" && newValue != "":
+				return false
+			default:
+				return true
+			}
+		},
+		DiffSuppressOnRefresh: true,
+	},
+	"initialize": {
```

Comment:
Please add validation for both refresh and initialize (you can check e.g. `table_constraint.go`).